Tests/termsAndConditionsTests.js

```diff
@@ -0,0 +1,17 @@
+var homePage = require('../Pages/homePage.js'),
+        
+    data = require("../Data/data.js");
+    
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+describe("006: Terms and Conditions", function (){
+    beforeEach(() => {
+        homePage.openPageURL(data.homepageLink)
+        })
+    it ("001: User is able to open 'Terms and Conditions' section", function(){
+        homePage.clickOnTermsAndConditions()
```

Comment:
It's located in page.js, and because page.js isn't exported in a way as other page objects, I can't perform page.method(), i have to use pageObject.method()